scripts/run-vim

```diff
@@ -23,11 +31,21 @@ if [ ! -f "$dir/bin/vim" ]; then
   exit 1
 fi
 
-$dir/bin/vim --noplugin -u NONE -N \
-  +"set shm+=WAFI rtp=$dir/share/vim/vimgo packpath=$dir/share/vim/vimgo,$vimgodir" \
-  +'filetype plugin indent on' \
-  +'packloadall!' \
-  "$@"
+if [ $coverage -eq 1 ]; then
+  covimerage -q run --report-file /tmp/vim-go-test/cov-profile.txt --append \
+    $dir/bin/vim --noplugin -u NONE -N \
+      +"set shm+=WAFI rtp=$dir/share/vim/vimgo packpath=$dir/share/vim/vimgo,$vimgodir" \
+      +'filetype plugin indent on' \
+      +'packloadall!' \
+      "$@"
+else
+  $dir/bin/vim --noplugin -u NONE -N \
+    +"set shm+=WAFI rtp=$dir/share/vim/vimgo packpath=$dir/share/vim/vimgo,$vimgodir" \
+    +'filetype plugin indent on' \
+    +'packloadall!' \
+    "$@"
+fi
```

Comment:
Would be nice to have the base command just once, and then prepend the `covimerage` part in case of `coverage=1` - might help with keeping it in sync.

You could use `${coverage:+covimerage -q run --report-file /tmp/vim-go-test/cov-profile.txt --append}` - and then have `$coverage` be empty/unset without `-c`.